data/json/items/gun/cannon.json

```
@@ -0,0 +1,30 @@
+[
+  {
+    "id": "cannon_3in_ordnance",
+    "looks_like": "remington_870",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "name": { "str": "3-inch ordnance rifle", "str_pl": "3-inch ordnance rifles" },
+    "description": "A cannon widely used in the American Civil War, noted for its reliability.",
```

```suggestion
    "description": "A light cannon widely used in the American Civil War, noted for its reliability.",
```
As field guns go, this was pretty much the smallest.